css/style.css

```diff
@@ -159,29 +152,25 @@ header h2 {
   justify-content: space-between;
 }
 
-.navbar-icon {
-  border: none;
-}
-
-.navbar ul {
-  display: none;
-}
-
-.navbar li {
-  align-items: center;
-  justify-content: space-between;
+.navbar .logo {
+  color: var(--secondary-color);
 }
 
 .navbar a {
   color: #344563;
   padding: 20px;
   margin: 0 5px;
-  line-height: 20px;
-  font-size: 15px;
 }
 
-.nav-list a:hover {
-  border-bottom: 2px #2a9df4 solid;
+.menu-btn {
```

Comment:
- [ ] Your menu button has no padding/margin to the right, please consider adding some space so your design is as close as possible to the Figma one

- Your design:
![Screenshot_20220510_220930](https://user-images.githubusercontent.com/25227022/167722285-996a4d2b-1650-43fd-ba33-7d86a6e13f8e.png)

- Figma design:
![Screenshot_20220510_221023](https://user-images.githubusercontent.com/25227022/167722355-cbd7a325-09ef-4978-96ef-4d56d0b3ea70.png)